src/bridge/tests/token_actions_test.cairo

```diff
@@ -217,3 +333,113 @@ fn consume_message_zero_recipient() {
     mock.consume_message(usdc_address, 100, contract_address_const::<0>());
 }
 
+
+#[test]
+fn send_deploy_message_ok() {
+    let mut mock = mock_state_testing();
+    let usdc_address = deploy_erc20("USDC", "USDC");
+
+    // Deploy messaging mock with 5 days cancellation delay
+    let messaging_mock_class_hash = snf::declare("messaging_mock").unwrap();
+    // Deploying with 5 days as the delay time (5 * 86400 = 432000)
+    let (messaging_contract_address, _) = messaging_mock_class_hash
+        .deploy(@array![DELAY_TIME])
+        .unwrap();
+
+    snf::start_cheat_caller_address_global(snf::test_address());
+    TokenBridge::constructor(ref mock, L3_BRIDGE_ADDRESS(), messaging_contract_address, OWNER());
+
+    mock.send_deploy_message(usdc_address);
+}
+
+#[test]
+#[should_panic(expected: ('L3 bridge not set',))]
+fn send_deploy_message_bridge_unset() {
+    let mut mock = mock_state_testing();
+    let usdc_address = USDC_MOCK_ADDRESS();
+
+    mock.send_deploy_message(usdc_address);
+}
+
+#[test]
+fn send_deposit_message_ok() {
+    let mut mock = mock_state_testing();
+    let usdc_address = USDC_MOCK_ADDRESS();
+
+    // Deploy messaging mock with 5 days cancellation delay
+    let messaging_mock_class_hash = snf::declare("messaging_mock").unwrap();
+    // Deploying with 5 days as the delay time (5 * 86400 = 432000)
+    let (messaging_contract_address, _) = messaging_mock_class_hash
+        .deploy(@array![DELAY_TIME])
+        .unwrap();
+
+    TokenBridge::constructor(ref mock, L3_BRIDGE_ADDRESS(), messaging_contract_address, OWNER());
+
+    let no_message: Span<felt252> = array![].span();
+    mock
```

Comment:
we should check if the message was received

Reply:
updated